src/main/java/net/fabricmc/loom/extension/LoomGradleExtensionImpl.java

```diff
@@ -67,6 +71,7 @@ public LoomGradleExtensionImpl(Project project, LoomFiles files) {
 		this.mixinApExtension = project.getObjects().newInstance(MixinApExtensionImpl.class, project);
 		this.loomFiles = files;
 		this.unmappedMods = project.files();
+		this.forgeExtension = Suppliers.memoize(() -> isForge() ? project.getObjects().newInstance(ForgeExtensionImpl.class, project) : null);
```

Comment:
Hmm, this is a bit ugly... It might be just nicer to initialise it in `getForge`?